### PR TITLE
Changed home button to show Fuel Quote page

### PR DESCRIPTION
**Pull request by tnttruong**
image

### CHANGES
- #23 
     - Clicking the home button now brings user directly to the fuel quote form, getting rid of the extra headers
- #29 
     - Generate Fuel Quote page has been removed since its functionally the "Home" page now

### ISSUES
- CSS is messed up on the right aligned buttons

### CODE REVIEW

File: src/backend/src/main.rs [hunk 1]
```diff
@@ -40,4 +40,5 @@ async fn rocket() -> _ {
         .mount("/static", FileServer::from("./static"))
         .manage(database)
         .manage(app_config)
-}
\ No newline at end of file
+}
+
```
Comment (Brod8362) [inline src/backend/src/main.rs:44]: Be careful not to check in pointless changes.

File: src/backend/templates/home.html.tera [hunk 2]
```diff
@@ -11,11 +11,9 @@
     {% if user %}
     <ul>
         <h1>Hello, {{user}}</h1>
-        <li><a href="#home">Home</a></li>
-        <li><a hx-get="/page/quote/history" hx-swap="innerHTML" hx-target="#title-menu">Fuel Quote History</a></li>
-        <li><a hx-get="/page/quote" hx-swap="innerHTML" hx-target="#title-menu">Generate Fuel Quote</a></li>
-        <li><a hx-get="/page/profile" hx-swap="innerHTML" hx-target="#title-menu">Profile</a></li>
-        <li><a hx-get="/logout" hx-swap="innerHTML" hx-target="#title-menu"></a></li>
+        <li><a hx-get="/page/quote/history" hx-swap="innerHTML" hx-target="#title-menu">Home</a></li>
```
Comment (Brod8362) [inline src/backend/templates/home.html.tera:14]: Clicking "home" and bringing the user to the fuel quote page does not make sense - the button label does not match what you expect it to do

File: src/backend/templates/login_page.html.tera [hunk 3]
```diff
@@ -35,7 +35,11 @@
 
             {% if message %}
             <div class="message">
+                
                 <p>{{message}}</p>
+                <script>
+                    location.reload();
+                </script>
```
Comment (Brod8362) [inline src/backend/templates/login_page.html.tera:42]: No JS.

**Review by Brod8362 — CHANGES_REQUESTED** [review]
See changes requested. Also, the screenshot in the PR indicates a problem with the layout (why are there scroll bars in the header?)

**Comment by Brod8362** [comment]
Also, you should probably rebase this on main if it isn't already, the commit history is a mess

File: src/backend/static/styles.css [hunk 4]
```diff
@@ -43,24 +42,16 @@ td {
...
-    color: white;
-}
-
 /* MENU HEADER */
 ul{
     list-style-type: none;
     margin: 0;
     padding: 0px;
-    overflow:hidden;
+    overflow: scroll;
```
Comment (Mnguyener) [inline src/backend/static/styles.css:50]: Line 50 is most likely the reason why the tabs are scrolling. We can put it back to `overflow: hidden` until we find another alternative to resizing windows

File: src/backend/templates/login_page.html.tera (hunk 3 above)
Comment (Mnguyener) [reply src/backend/templates/login_page.html.tera:42]: I believe we can do this by setting `HX-Refresh` to `true`. [More on that](https://htmx.org/reference/#response_headers)
and [Page reload question](https://www.reddit.com/r/htmx/comments/14ca5wn/page_reload_button_with_htmx_no_swapping_required/)

**Review by Mnguyener — CHANGES_REQUESTED** [review]
Let me know if you need help on the refresh page mechanic with HTMX

**Comment by Mnguyener** [comment]
> See changes requested. Also, the screenshot in the PR indicates a problem with the layout (why are there scroll bars in the header?)

**Comment by tnttruong** [comment]
closing PR to redo branch since checkout to branch with same name messed up a bunch of stuff

**Pull request closed without merging**